src/tarball.py

```diff
@@ -121,7 +130,8 @@ def run(self):
                     build_config.tree.name, self._kdir
                 )
                 tarball = self._push_tarball(build_config, describe)
-                self._update_node(node, describe, tarball, "pass")
+                self._update_node(node, describe, tarball)
+                self._create_node(node, "pass")
```

Comment:
I think the method names should be changed to make it clear what kind of node is being updated or created. For example, `_create_tarball_node()` and `_update_revision_node()`.

Reply:
Renamed.